src/common/components/repository-row/dropdowns/register-name-dropdown.js

```diff
@@ -231,9 +231,13 @@ const ActionButtons = (props) => {
 
   return (
     <div className={ styles.buttonRow }>
-      <a onClick={onCancelClick} className={ styles.cancel }>
+      <Button
+        appearance="base"
+        onClick={onCancelClick}
+      >
         Cancel
-      </a>
+      </Button>
+      {' '}
```

Comment:
This makes me think of the very complex button rules we had for button groups (in a pre Vanilla CSS lib), but you could obviously control this with `button + button:not(:last-child)` kind of thing.